tests/test_io.py

```diff
@@ -137,7 +137,7 @@ def test_convenience_functions(self, method, entry_point_name, mock_plugin_conve
     def test_from_qasm(self, mock_plugin_converters):
         """Tests that the correct entry point is called for from_qasm."""
 
-        qml.from_qasm("Test", measurements=None)
+        qml.from_qasm("Test")
```

Comment:
Sorry, maybe I'm not getting the `mock_plugin_converters` logic or something, but are we checking that no warning is raised? :sweat_smile:

Reply:
So the functions in PL calls their corresponding functions in the plugins. `mock_plugin_converters` mocks the `load` functions defined in the plugins, it does not mock the PL side, which is where the warning would be raised.

Reply:
Thanks, I got it now! I was thinking that - because I wasn't aware of the Deprecation warning being considered an error - the mocking would do this part, but that's not the case I understand now :)